flow/TLSConfig.actor.cpp

```diff
@@ -211,6 +221,15 @@ std::string TLSConfig::getCAPathSync() const {
 	return envCAPath;
 }
 
+bool TLSConfig::getDisablePlainTextConnection() const {
+	std::string envDisablePlainTextConnection;
+	if (platform::getEnvironmentVar("FDB_TLS_DISABLE_PLAINTEXT_CONNECTION", envDisablePlainTextConnection)) {
+		return boost::lexical_cast<bool>(envDisablePlainTextConnection);
```

Comment:
I don't see why we should read EnvVar in both places (`LoadedTLSConfig` and `TLSConfig`) seeing as this variable doesn't change for the duration of the process. Loading it once in `TLSConfig` and forwarding it to `LoadedTLSConfig` should be enough. `LoadedTLSConfig` can refer to the cached boolean.